### ISSUE TITLE
I can't open the settings if the volume is open

### ISSUE THREAD

**Issue by OmlineEditor** [hardcore-sushi/DroidFS#72]
I ask you to make it possible to open the application settings when the encrypted volume is open. you can't do this now, you need to close the volume and only then you can get into the settings

**Comment by hardcore-sushi** [comment]
That's true, but we need to take care of closing the volume if the user leaves the app while he's on the settings page.

**Comment by OmlineEditor** [comment]
a pop-up notification in the status bar with the close container button will do, I think

**Comment by hardcore-sushi** [comment]
No, we want to close the container as soon as the user leaves the app (except if the "Keep volume open" unsafe feature is enabled of course). Otherwise the app could be killed by the system while in the background, and therefore the volume wouldn't be closed properly.

**Comment by OmlineEditor** [comment]
you can make an additional way to close the application through a pop-up notification in the status bar if the "Keep volume open" function is not installed safely

**Comment by hardcore-sushi** [comment]
Yes but that's another topic.

**Comment by hardcore-sushi** [comment]
I won't do it because I have many other tasks to do before this, so it's as you like.

**Comment by OmlineEditor** [comment]
maybe in the future you will make this feature

**Comment by hardcore-sushi** [comment]
Fixed with 1a1d3ea570781ed12472bb7378bf4b93e26b78f6.
